Dockerfile

```diff
@@ -4,14 +4,14 @@ FROM golang:1.19-alpine3.16 AS go-builder
 WORKDIR /src
 
 # CosmWasm: see https://github.com/CosmWasm/wasmvm/releases
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.2.1/libwasmvm_muslc.aarch64.a /lib/libwasmvm_muslc.aarch64.a
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.2.1/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.2.3/libwasmvm_muslc.aarch64.a /lib/libwasmvm_muslc.aarch64.a
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.2.3/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
```

Comment:
We should bump again to the fresh [v1.2.4](https://github.com/CosmWasm/wasmvm/releases/tag/v1.2.4) version to prevent incompatibility issues between compiled wasm modules in the cache, see: https://github.com/CosmWasm/wasmvm/issues/426.

The `wasm` module should be upgraded as well to [v0.40.1](https://github.com/CosmWasm/wasmd/releases/tag/v0.40.1) 😉